docs/docs/README.md

```diff
@@ -0,0 +1,581 @@
+
+WFL uses a `gcloud auth` command line to authenticate the user. You
+need to be authenticated to Google Cloud and have a recent version
+of `google-cloud-sdk` in your path to run `zero` or its jar
+successfully. I verified that `Google Cloud SDK 161.0.0` works. That
+or any later version should be OK.
+
+1.  Cheatsheets
+    
+    I find a cheatsheet handy when programming in Clojure. There are
+    a bunch. Bookmark or print one.
+    
+      - <https://clojure.org/api/cheatsheet>
+      - <https://www.conj.io/> (… used to be called Grimoire …)
+      - <https://github.com/jafingerhut/clojure-cheatsheets/tree/master/pdf>
+      - <http://cljs.info/cheatsheet/> (ClojureScript)
+      - <https://github.com/jafingerhut/clojure-cheatsheets>
+        (sources)
+    
+    These may also be handy.
+    
+      - [Clojure Error
+        Messages](https://github.com/yogthos/clojure-error-message-catalog)
+      - ["Weird" Characters in
+        Clojure](https://clojure.org/guides/weird_characters)
+
+2.  Installation
+    
+    See [this link](https://github.com/boot-clj/boot#install) to
+    install `boot-clj`.
+    
+    Running `boot` is enough to "install" Clojure.
+    
+    There is another tool like `boot` named `lein`, which is short
+    for "[Leiningen](https://leiningen.org/)". You currently need
+    `lein` to develop with
+    [IntelliJ](https://www.jetbrains.com/idea/) using its Clojure
+    plugin [Cursive](https://cursive-ide.com/).
+    
+    On MacOS, I suggest installing [Homebrew](http://brew.sh/) and
+    then running this.
+    
+    ``` bash
+    zero # brew install boot-clj leiningen
+    ==> Using the sandbox
+    ==> Downloading https://github.com/boot-clj/boot-bin/releases/download/2.5.2/boot
+    🍺  /usr/local/Cellar/boot-clj/2.5.2: 3 files, 7.7K, built in 2 seconds
+    ...
+    zero #
+    ```
+    
+    You can `brew install maven`, and `java` too if necessary.
+    
+    There are `boot-clj` and `lein` distributions for all the common
+    OS platforms. Each tool is just a file. Copy them into your
+    `PATH`, run them once to bootstrap them, and you're done. (The
+    first run of each tool downloads dependencies and so on.)
+    
+    The `build.boot` file is equivalent to the `build.sbt` file for
+    SBT in Scala projects. It specified project dependencies and the
+    build and release pipeline. It also functions as a script for
+    running and testing the project without a separate compilation
+    step.
+
+3.  Hacking
+    
+    Clojure development feels very different from Scala and Java
+    development. It even differs markedly from development in other
+    *dynamic languages* such as Python or Ruby.
+    
+    Get a demonstration from someone familiar with Clojure
+    development before you spend too much time trying to figure
+    things out on your own.
+    
+    Find a local Cursive user for guidance if you like IntelliJ.
+    [Jay Carey](mailto:jcarey@broadinstitute.org) and [Charley
```

Comment:
Both of these people no longer work at the Broad.

Reply:
I will update this.